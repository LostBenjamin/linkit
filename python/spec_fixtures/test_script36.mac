
$num-instances 6
$sched-time 50

[app 10]
    [[[sched `1*<num-instances>` `20+INSTANCE*<num-instances>`]]]

(((object <num-instances>)))

[[object INSTANCE
  [sched-INSTANCE]
    INSTANCE
    red
]]

