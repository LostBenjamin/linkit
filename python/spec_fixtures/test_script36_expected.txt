10:set:11:run
4:set:0:red
5:set:1:red
6:set:2:red
7:set:3:red
8:set:4:red
9:set:5:red
11:set:20,4:sch:26,5:sch:32,6:sch:38,7:sch:44,8:sch:50,9:sch

